### PR TITLE
Uddate API docs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2560
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by daneshk — APPROVED** [review]

**Comment by daneshk** [comment]
@praneesha can you check and approve

File: ballerina/file_errors.bal [hunk 1]
```diff
@@ -23,31 +23,31 @@ public type PermissionError distinct Error;
...
-# Represents an error that occurs when the file/directory does not exist at the given filepath.
+# Represents an error that occurs when the file/directory does not exist at the given file path.
 public type FileNotFoundError distinct Error;
 
-# Represents error occur when the file at the given filepath is not a symbolic link.
+# Represents error occurs when the file at the given file path is not a symbolic link.
 public type NotLinkError distinct Error;
 
-# Represents IO error occur when trying to access the file at the given filepath.
+# Represents IO error occur when trying to access the file at the given file path.
```
Comment (praneesha) [inline ballerina/file_errors.bal:32]:
```suggestion
# Represents an IO error, which occurs when trying to access the file in the given file path.
```

File: ballerina/file_errors.bal [hunk 2]
```diff
@@ -23,31 +23,31 @@ public type PermissionError distinct Error;
...
-# Represents security error occur when trying to access the file at the given filepath.
+# Represents security error occur when trying to access the file at the given file path.
 public type SecurityError distinct Error;
 
-# Represents error occur when the given file path is invalid.
+# Represents error occurs when the given file path is invalid.
 public type InvalidPathError distinct Error;
 
-# Represent error occur when the given pattern is not a valid filepath pattern.
+# Represent error occurs when the given pattern is not a valid file path pattern.
```
Comment (praneesha) [inline ballerina/file_errors.bal:41]:
```suggestion
# Represents an error, which occurs when the given pattern is not a valid file path pattern.
```

File: ballerina/service_endpoint.bal [hunk 3]
```diff
@@ -72,7 +72,7 @@ public isolated class Listener {
     }
 }
 
-# Represents configurations that required for directory listener.
+# Represents configurations that are required for a directory listener.
```
Comment (praneesha) [inline ballerina/service_endpoint.bal:75]:
```suggestion
# Represents the configurations that are required for a directory listener.
```

**Review by daneshk — APPROVED** [review]

**Pull request merged**